resources/victron-virtual-functions.js

```diff
@@ -475,26 +477,17 @@
         const rgbCheckboxes = $(`
         <div id="switch-1-rgb-checkboxes" style="margin-top:10px;">
           <label style="font-weight:bold;">Select RGB control types (at least one required):</label>
-          <div class="form-row">
-            <label>&nbsp;</label>
-            <label for="node-input-switch_1_rgb_color_wheel" style="width:70%;">
-              <input type="checkbox" id="node-input-switch_1_rgb_color_wheel" style="display:inline-block; width:22px; vertical-align:baseline;">
-              RGB color wheel
-            </label>
+          <div class="form-row victron-checkbox">
+            <input type="checkbox" id="node-input-switch_1_rgb_color_wheel">
+            <label for="node-input-switch_1_rgb_color_wheel">RGB color wheel</label>
```

Comment:
Something in the layout broke here. The documentation seems to be indented unnecessarily, wasting space (and possibly causing issue son narrow displays).

This is master:

<img width="491" height="368" alt="Image" src="https://github.com/user-attachments/assets/40c24637-87ff-4908-b7b3-9be77b2c1a42" />

... and this is this PR:

<img width="727" height="478" alt="Image" src="https://github.com/user-attachments/assets/943d184c-3838-4789-a4f9-d09b94221973" />